.github/workflows/scheduled.yaml

```diff
@@ -13,34 +13,8 @@ jobs:
       fail-fast: false
       matrix:
         DEB_DISTRO: [jammy, noble]
-    uses: ubi-agni/ros-builder-action/.github/workflows/build.yaml@main
+        ARCH: [x64, arm64]
+    uses: ./.github/workflows/scheduled_job.yaml
```

Comment:
Why did you factor out the actual build rules? scheduled_job.yaml is not used elsewhere, is it?

Reply:
From what I've researched, it seems that the function to execute the `deploy` job upon success for each of the matrix-ized Jobs in Github Actions has not been implemented.
Therefore, I've decided to create a set of the jobs for each element of the matrix (`build`) and the `deploy` job as a reusable workflow, and then put that workflow job into a matrix.

Reply:
ref. https://github.com/orgs/community/discussions/26639